### ISSUE TITLE
@author is to long

### ISSUE THREAD

**Issue by stevleibelt** [bazzline/php_propel_behavior_entity_instantiator#2]
When I create a file, I get something like thins in the comments.

```
/**
 * Class <file name>
 *
 * @author Net\Bazzline\Propel\Behavior\EntityInstantiatorNet\Bazzline\Propel\Behavior\EntityInstantiator\FileContentGenerator
 * @since 2015-11-24
 * @see http://www.bazzline.net
 */
```

I don't think that this is expected ;-).

**Comment by stevleibelt** [comment]
Started working on this in branch [issue_2](https://github.com/bazzline/php_propel_behavior_entity_instantiator/tree/issue_2).

**Comment by stevleibelt** [comment]
Fixed within branch [issue_2](https://github.com/bazzline/php_propel_behavior_entity_instantiator/tree/issue_2) and will be available in release [1.1.0](https://github.com/bazzline/php_propel_behavior_entity_instantiator/releases/tag/1.1.0).
